ACE/ace/OS_NS_Thread.inl

```diff
@@ -218,31 +218,18 @@ ACE_OS::condattr_synctype (ACE_condattr_t &attributes, int& type)
 ACE_INLINE int
 ACE_OS::condattr_setclock (ACE_condattr_t &attributes, clockid_t clock_id)
 {
-# if defined (ACE_HAS_THREADS)
```

Comment:
Why remove the check for ACE_HAS_THREADS? We can build ACE without thread support

Reply:
I know that, this implementation is only active if `ACE_CONDATTR_SETCLOCK` is defined in `config-posix.h`, which in turn is ultimately defined if `ACE_HAS_THREADS` is defined.